src/coprocessor/dag/expr/mod.rs

```diff
@@ -66,6 +66,13 @@ pub struct ScalarFunc {
     cus_rng: CusRng,
 }
 
+impl ScalarFunc {
+    /// Indicates whether the current expression is evaluated in union statement
+    pub fn in_union(&self) -> bool {
+        self.implicit_args.get(0) == Some(&Datum::I64(1))
```

Comment:
Is there any doc mentioning the first argument being "1" means "in union"? If yes please add a link.